index.html

```diff
@@ -4,5 +4,9 @@
   <title>comroid.de</title>
 </head>
 <body>
-  <iframe src="https://discordapp.com/widget?id=297846944754040842&theme=dark" width="350" height="500" allowtransparency="true" frameborder="0"></iframe>
```

Comment:
Please try to implement this embed somehow, doesn't have to be nice, but at least it should be visible. The embed is already configured.

Reply:
There's this href for. 
We can't iframe discord.

Reply:
Tho this iframe is a 1:1 template issued by discord

Reply:
This might be true for the main website but does this apply to the invite site too?

Reply:
No. That one shouldn't be embedded. This was the already changed embed 😜